modin/pandas/test/test_groupby.py

```diff
@@ -1111,6 +1119,55 @@ def eval___getattr__(modin_groupby, pandas_groupby, item):
     )
 
 
+def eval__getitem__(md_grp, pd_grp, item):
```

Comment:
Other `eval_*` functions seem to follow the pattern of having three underscores (one from `eval_` and two from a special dunder method), I suggest we keep the pattern.

Reply:
aligned